.github/workflows/gherkin-lint.yml

```diff
@@ -0,0 +1,46 @@
+name: Gherkin Linting
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    # Only run if PHP-related files changed.
+    paths:
+      - '.github/workflows/behat-test.yml'
+      - '**.php'
+      - '**.feature'
+      - 'behat.yml'
+      - 'composer.json'
+      - 'composer.lock'
```

Comment:
This should only really include the `.feature` files, no?

Reply:
I copied those from `behat-test.yml`. I will refine those.